sql/wsrep_trans_observer.h

```diff
@@ -345,6 +353,12 @@ static inline int wsrep_before_commit(THD* thd, bool all)
     wsrep_register_for_group_commit(thd);
   }
 
+  WSREP_DEBUG("wsrep_before_commit: %d seqno=%lld for %s ret=%s",
```

Comment:
Again, are these logging changes necessary? As in `wsrep_before_prepare()`, I don't see the relevance and there is high chance of getting merge conflicts.

Reply:
Removed.